std_msgs/Header header

uint8 horiz_mode
uint8 vert_mode
uint8 yaw_mode
uint8 level_frame
uint8 torsion_frame

dji_onboard_sdk/api_cmd_data ctrl_data
